onvm/onvm_nflib/onvm_nflib.c

```diff
@@ -1290,3 +1294,34 @@ init_shared_core_mode_info(uint16_t instance_id) {
 
         nf->shared_core.sleep_state = (rte_atomic16_t *)shm;
 }
+
+void
+onvm_nflib_stats_summary_print(uint16_t id) {
+        const char clr[] = {27, '[', '2', 'J', '\0'};
+        const char topLeft[] = {27, '[', '1', ';', '1', 'H', '\0'};
+        const uint64_t rx = nfs[id].stats.rx;
+        const uint64_t rx_drop = nfs[id].stats.rx_drop;
+        const uint64_t tx = nfs[id].stats.tx;
+        const uint64_t tx_drop = nfs[id].stats.tx_drop;
+        const uint64_t act_out = nfs[id].stats.act_out;
+        const uint64_t act_tonf = nfs[id].stats.act_tonf;
+        const uint64_t act_drop = nfs[id].stats.act_drop;
+        const uint64_t act_next = nfs[id].stats.act_next;
+        const uint64_t act_buffer = nfs[id].stats.tx_buffer;
+        const uint64_t act_returned = nfs[id].stats.tx_returned;
+
+        /* Clear screen and move to top left */
+        printf("%s%s", clr, topLeft);
+        printf("NF activity summary\n");
+        printf("----------------------------------------------------\n");
+        printf("RX tot: %ld\n", rx);
```

Comment:
Maybe expand the tot-> total? As nf terminated we can use more screen space and it would look nicer.